manual/cypher/cypher-docs/src/docs/dev/clause.asciidoc

```diff
@@ -0,0 +1,157 @@
+[[query-clause]]
+= Clauses
+
+[discrete]
```

Comment:
I don't think the AsciiDoc processor knows `[discrete]` so it chokes. Can we make do without it?